dolphinscheduler-dao/src/test/java/org/apache/dolphinscheduler/dao/mapper/ProcessInstanceMapperTest.java

```diff
@@ -51,6 +51,23 @@
     @Autowired
     ProjectMapper projectMapper;
 
+    /**
+     * insert process instance with specified start time and end time,set state to SUCCESS
+     * @param startTime
+     * @param endTime
+     * @return
+     */
+    private ProcessInstance insertOne(Date startTime,Date endTime){
```

Comment:
```suggestion
    private ProcessInstance insertOne(Date startTime, Date endTime) {
```

Please configure the checkstyle.xml plugin[1] to your IDE to format code.
[1] https://github.com/apache/incubator-dolphinscheduler/tree/dev/style

Reply:
ok , I will fix it